script.js

```diff
@@ -1,4 +1,141 @@
+//Game: Beat That!
+//2 players, 2 dices
+//2 game modes: Each player rolls once and player picks the order of the numbers rolled to get the biggest value
+//Player with higher combined number wins
+//Calculate number of wins
+
+//2 players
+var player1 = "Player 1";
+var player2 = "Player 2";
+var player1DiceRoll = [];
+var player2DiceRoll = [];
+var player1Number;
+var player2Number;
+var player1WinCount = 0;
+var player2WinCount = 0;
+var player1LoseCount = 0;
+var player2LoseCount = 0;
+var drawCount = 0;
```

Comment:
great usage of global variables here!

But there are slightly redundant variables here. because player1WinCount will always be exactly the same as player2LoseCount, and vice versa.